src/evo/netinfo.h

```diff
@@ -146,11 +175,12 @@ class NetInfoInterface
 public:
     virtual ~NetInfoInterface() = default;
 
-    virtual NetInfoStatus AddEntry(const std::string& service) = 0;
+    virtual NetInfoStatus AddEntry(const uint8_t purpose, const std::string& service) = 0;
     virtual NetInfoList GetEntries() const = 0;
 
     virtual CService GetPrimary() const = 0;
     virtual bool CanStorePlatform() const = 0;
+    virtual bool HasEntries(uint8_t purpose) const = 0;
```

Comment:
consider passing here enum instead uint8_t

Reply:
See [comment](https://github.com/dashpay/dash/pull/6666#discussion_r2286472412), resolved in latest push.